gradle.properties

```diff
@@ -0,0 +1 @@
+android.useAndroidX=true
```

Comment:
This file needs a newlines at the end. As assistance for others in a similar situation, how about adding the following `.editorconfig` file?

```
root = true

[*]
end_of_line = lf
insert_final_newline = true

charset = utf-8

[*.java]
indent_style = tab
```


It depends on tooling for having it respected, but most probably use Android Studio, which will respect it... actually while checking that, I see that tool even has an option to preview editorconfig settings live to see if they work as desired. Pretty cool.

Reply:
Done.